### PR TITLE
Update documentation about docker-compose

### PR DESCRIPTION
**Pull request by jmleroux**
Related to this proposal: https://github.com/akeneo/pim-community-dev/pull/7895

### CODE REVIEW

File: install_pim/docker/installation_docker.rst [hunk 1]
```diff
@@ -48,9 +48,11 @@ or from our partner portal if you have access to the enterprise edition. It can
 Using the Docker images
 -----------------------
 
-Every flavor (dev or standard, community or enterprise) comes with a Docker Compose file template ``docker-compose.yml.dist``, ready to be used.
-Copy it as ``docker-compose.yml`` and keep it at the root of your project. You may modify it at your convenience, to change the mapping of the ports
-if you want Apache to be accessible from a port other that 8080, for instance.
+Every flavor (dev or standard, community or enterprise) comes with a Docker Compose file ``docker-compose.yml`` and a ``docker-compose.override.yml.dist``, ready to be used. The ``docker-compose.yml`` file configure all the necessary containers with the defaults settings to run Akeneo PIM. The ``docker-compose.override.yml.dist`` defines the custom parameters you may change depending on your development environment.
```
Comment (tamarasaurus) [inline install_pim/docker/installation_docker.rst]: configure --> configure**s**

Comment (tamarasaurus) [reply install_pim/docker/installation_docker.rst]: defaults --> default

File: install_pim/docker/installation_docker.rst [hunk 2]
```diff
@@ -48,9 +48,11 @@ or from our partner portal if you have access to the enterprise edition. It can
...
+
+Copy the override file as ``docker-compose.override.yml`` and keep it at the root of your project. You may modify it at your convenience. It defines some ports mapping by default and you can change them if you need, if you want Apache to be accessible from a port other that 8080, for instance.
+
+However, **you need at least one mapping for ``fpm`` and ``fpm-test`` containers**. We should set the default mapping in the root ``docker-compose`` file, but it is currently not possible to repplace array values in the override. You can read more here: https://docs.docker.com/compose/extends/#adding-and-overriding-configuration.
```
Comment (tamarasaurus) [inline install_pim/docker/installation_docker.rst]: repplace --> replace

Comment (damien-carcel) [reply install_pim/docker/installation_docker.rst]: ``fpm`` and ``fpm-test`` => akeneo and akeneo-behat?

File: install_pim/docker/installation_docker.rst (hunk 1 above)
Comment (jmleroux) [reply install_pim/docker/installation_docker.rst]: Thx @tamarasaurus :+1:

File: install_pim/docker/installation_docker.rst [hunk 3]
```diff
@@ -48,9 +48,11 @@ or from our partner portal if you have access to the enterprise edition. It can
...
-if you want Apache to be accessible from a port other that 8080, for instance.
+Every flavor (dev or standard, community or enterprise) comes with a Docker Compose file ``docker-compose.yml`` and a ``docker-compose.override.yml.dist``, ready to be used. The ``docker-compose.yml`` file configures all the necessary containers with the default settings to run Akeneo PIM. The ``docker-compose.override.yml.dist`` defines the custom parameters you may change depending on your development environment.
+
+Copy the override file as ``docker-compose.override.yml`` and keep it at the root of your project. You may modify it at your convenience. It defines some ports mapping by default and you can change them if you need, if you want Apache to be accessible from a port other that 8080, for instance.
+
+However, **you need at least one mapping for ``fpm`` container**. We should set the default mapping in the root ``docker-compose`` file, but it is currently not possible to repplace array values in the override. You can read more here: https://docs.docker.com/compose/extends/#adding-and-overriding-configuration.
```
Comment (damien-carcel) [inline install_pim/docker/installation_docker.rst]: Nope. It is `httpd` and `httpd-behat` that needs a mapping in 2.x and 3.x. Those are Apache containers. FPM doesn't need at all.

Comment (jmleroux) [reply install_pim/docker/installation_docker.rst]: Mais oui ! Chuis con des fois :smile:

**Comment by jmleroux** [comment]
@damien-carcel and others, I need approval of the PIM PR before merging this one.

File: install_pim/docker/installation_docker.rst [hunk 4]
```diff
@@ -48,9 +48,11 @@ or from our partner portal if you have access to the enterprise edition. It can
 Using the Docker images
 -----------------------
 
-Every flavor (dev or standard, community or enterprise) comes with a Docker Compose file template ``docker-compose.yml.dist``, ready to be used.
-Copy it as ``docker-compose.yml`` and keep it at the root of your project. You may modify it at your convenience, to change the mapping of the ports
-if you want Apache to be accessible from a port other that 8080, for instance.
+Every flavor (dev or standard, community or enterprise) comes with a Docker Compose file ``docker-compose.yml`` and a ``docker-compose.override.yml.dist``, ready to be used. The ``docker-compose.yml`` file configures all the necessary containers with the default settings to run Akeneo PIM. The ``docker-compose.override.yml.dist`` defines the custom parameters you may change depending on your development environment.
+
+Copy the override file as ``docker-compose.override.yml`` and keep it at the root of your project. You may modify it at your convenience. It defines some ports mapping by default and you can change them if you need, if you want Apache to be accessible from a port other that 8080, for instance.
```
Comment (damien-carcel) [inline install_pim/docker/installation_docker.rst]: Beware, for now there is no `docker-compose.override.yml.dist` in your pim-community-dev PR.

**Comment by damien-carcel** [comment]
I just realized you also need to add some stuff in the "debug" section. Some mappings and environment variables were to be changed before, now they have to be added instead in the override.

**Pull request merged**